src/libcore/cmp.rs

```diff
@@ -8,35 +8,33 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
-//! Defines the `PartialOrd` and `PartialEq` comparison traits.
+//! Traits for ordering and comparison.
```

Comment:
Maybe s/Traits/Functionality/ since there's things other than traits? (I could just be overly nit-picky)